widgets/dialogs.pas

```diff
@@ -278,6 +287,33 @@ procedure TMessageDialog.KeyDown(var Key: NativeInt; Shift: TShiftState);
   end;
 end;
 
+procedure TMessageDialog.Show;
+var
+  ownForm: TWForm;
+  curHeight: Integer;
+begin
+  inherited Show;
+  if not (Owner is TWForm) then
+    exit;
+  ownForm := TWForm(Owner);
```

Comment:
I think that doesn't belong here considering you had done that further down

Reply:
Thanks, at the time of this writing I got distracted and the code was duplicated. 
Fixed.